tools/TestFx/Live/InvokeLiveTestScenarios.ps1

```diff
@@ -1,66 +1,206 @@
 param (
     [Parameter(Mandatory)]
     [ValidateNotNullOrEmpty()]
-    [string] $RunPlatform,
+    [string] $OSVersion,
 
     [Parameter(Mandatory)]
     [ValidateNotNullOrEmpty()]
-    [string] $RunPowerShell,
+    [string] $RunPlatform,
 
     [Parameter(Mandatory)]
     [ValidateNotNullOrEmpty()]
-    [string] $PowerShellLatest,
+    [string] $RunPowerShell
+)
 
-    [Parameter(Mandatory)]
-    [ValidateScript({ Test-Path -LiteralPath $_ -PathType Container })]
-    [string] $RepoLocation,
+$srcDir = Join-Path -Path ${env:BUILD_SOURCESDIRECTORY} -ChildPath "src"
+$liveScenarios = Get-ChildItem -Path $srcDir -Directory -Exclude "Accounts" -ErrorAction SilentlyContinue | Get-ChildItem -Directory -Filter "LiveTests" -Recurse | Get-ChildItem -File -Filter "TestLiveScenarios.ps1" | Select-Object -ExpandProperty FullName
 
-    [Parameter(Mandatory)]
-    [ValidateScript({ Test-Path -LiteralPath $_ -PathType Container })]
-    [string] $DataLocation
-)
+$maxRunspaces = 9
+[void][int]::TryParse(${env:RSPTHROTTLE}, [ref]$maxRunspaces)
+$rsp = [runspacefactory]::CreateRunspacePool(1, $maxRunspaces)
+$rsp.CleanupInterval = [timespan]::FromHours(10)
```

Comment:
// runspace may be disposed when waiting for resources to be removed. default 15min

Reply:
Comment added.